csv_2_po.py

```diff
@@ -0,0 +1,60 @@
+import fire
```

Comment:
@badrihippo , are these files supposed to be in home directory?, or inside /dashboard ?

Reply:
They're unusual files so there's no fixed place for them...maybe we can leave them as-is for now? And eventually (in a future task, after getting it running properly) we can make it into a `manage.py` command.